src/components/GoogleMap.svelte

```diff
@@ -6,7 +6,6 @@
   import { Loader } from '@googlemaps/js-api-loader';
   import { createEventDispatcher, onMount } from 'svelte';
 
-  export let index;
```

Comment:
Unrelated, we didn't appear to be using `index` anywhere in either map component.